app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/CheckAgencyController.scala

```diff
@@ -59,13 +59,8 @@ class CheckAgencyController @Inject()
   }
 
   val showCheckAgencyStatus: Action[AnyContent] = AuthorisedWithSubscribingAgent {
```

Comment:
imho we should use here another builder, not forcing enrolments read from auth when not used